heroku/cli
# refactor(pg-v5): Move 'pg:backups:download` to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b2I4oYAE/view

This converts `pg:backups:download` to oclif and updates its associated tests.

### Testing

1) Pull down this branch
2) Run `./bin/run pg:backups:download -a APP` with an app that has a pg backup on it
3) Verify the command succeeds, saving `latest.dump` to your current directory and compare results with prod command
4) Repeat 2-3 with supplying a custom filename using the `-o` flag
